index.html

```diff
@@ -67,157 +67,170 @@ <h2 class="worksTitle">My Recent Works</h2>
+                or sign-ups required. Has been the industry's standard.
+              </p>
+            </div>
+            <ul class="mainListCategoris">
+              <li class="tags">HTML</li>
+              <li class="tags">Bootstrap</li>
+              <li class="tags">Ruby</li>
+            </ul>
           </div>
 
-          <ul class="mainListCategoris">
-            <li class="tags">HTML</li>
-            <li class="tags">Bootstrap</li>
-            <li class="tags">Ruby</li>
-          </ul>
-
           <button type="button" class="btn cardBtn">See project</button>
         </article>
       </div>
+
       <div class="works">
         <article class="worksCard">
-          <div class="cardContent">
-            <h2 class="CardText">Profesional Art Printing Data</h2>
-            <p class="paragraphCardText">
-              A daily selection of privately personalized reads; no accounts
-              or sign-ups required. Has been the industry's standard.
-            </p>
+          <div class="card-container">
+            <div class="cardContent">
+              <h2 class="CardText">Website Portfolio</h2>
+              <p class="paragraphCardText">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. Has been the industry's standard.
+              </p>
+            </div>
+            <ul class="mainListCategoris">
+              <li class="tags">HTML</li>
+              <li class="tags">Bootstrap</li>
+              <li class="tags">Ruby</li>
+            </ul>
           </div>
-
-          <ul class="mainListCategoris">
-            <li class="tags">HTML</li>
-            <li class="tags">Bootstrap</li>
-            <li class="tags">Ruby</li>
-          </ul>
-
           <button type="button" class="btn cardBtn">See project</button>
         </article>
       </div>
+
     </div>
   </section>
 
   <section id="aboutMe">
-    <div class="mainParagraph">
-      <h2 class="aboutMeText">About me</h2>
-      <p class="aboutMeDescription">
-        Hello I’m a software developer! I can help you build a product ,
-        feature or website Look through some of my work and experience! If you
-        like what you see and have a project you need coded, don’t hesitate to
-        contact me.
-      </p>
-      <button class="btn getResumeBtn" type="button">Get my resume</button>
+    <div class="aboutMeDiv">
+      <div class="mainParagraph">
+        <h2 class="aboutMeText">About me</h2>
+        <p class="aboutMeDescription">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hesitate to
+          contact me.
+        </p>
+        <button class="btn getResumeBtn" type="button">Get my resume</button>
+      </div>
```

Comment:
- [ ] I will encourage you to add space to the middle of the content so that you will get better at meticulously following designs.

### Yoyr design
![image](https://user-images.githubusercontent.com/86200356/236154228-d00a347d-c3ac-4d62-94da-445cde3c0d8f.png)


### Figma design
![image](https://user-images.githubusercontent.com/86200356/236154288-db2d7447-9d79-4b72-a948-dea784749426.png)